solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java

```diff
@@ -295,15 +296,12 @@ private static DocCollection collectionFromObjects(
   }
 
   @Override
-  public void write(JSONWriter jsonWriter) {
-    LinkedHashMap<String, DocCollection> map = new LinkedHashMap<>();
-    for (Entry<String, CollectionRef> e : collectionStates.entrySet()) {
-      if (e.getValue().getClass() == CollectionRef.class) {
```

Comment:
This if statement check was redundant before right?

Reply:
yes. it was redundant. I could remove it